LEAF_Request_Portal/sources/Workflow.php

```diff
@@ -529,20 +551,49 @@ public function updateStep($stepID, $stepTitle, $bgColor = '', $fontColor = '')
         }
 
         $vars = array(':stepID' => $stepID,
-                      ':stepTitle' => $stepTitle,
+            ':stepTitle' => $stepTitle,
         );
         $res = $this->db->prepared_query('UPDATE workflow_steps
     										SET stepTitle=:stepTitle
     										WHERE stepID=:stepID', $vars);
-        
+
         $this->dataActionLogger->logAction(\DataActions::MODIFY, \LoggableTypes::WORKFLOW_STEP, [
             new LogItem("workflows", "stepID", $stepID),
             new LogItem("workflows", "stepTitle",  $stepTitle),
             new LogItem("workflows", "jsSrc",  "", "empty"),
             new LogItem("workflow_steps", "workflowID", $this->getWorkflowIDFromStep($stepID))
-        ]);    
+        ]);
+
+        return 1;
+    }
+
+    public function saveStepData($stepID, $data){
```

Comment:
Need this function cleaned up.  Best practices including only a single return.  Would prefer a single return type as well.  Letting the calling function determine what to do with the int result.